src/config.rs

```diff
@@ -157,6 +157,13 @@ pub struct Config {
     /// local variable declaration. When set to false, the content is only availabe when
     /// holding the `ctrl` key in some editors.
     pub show_hover_context: bool,
+    /// EXPERIMENTAL (needs unstable features)
```

Comment:
It doesn't look like you're enforcing the 'unstable features' requirement

Reply:
I am resetting the option [here](https://github.com/rust-lang-nursery/rls/pull/988/files#diff-6d8a5e2104954a026de2bfe2b936aea4R232) if the unstable features are not enabled; am I missing something else?

Reply:
Are sorry, missed that. Looks fine.